command/teardown.go

```diff
@@ -27,12 +27,12 @@ func CmdTeardown(c *cli.Context) error {
 }
 
 func removeContainersWithoutGcLabel() error {
-	// only remove containers that don't have the com.lancope.docker-gc.keep set
+	// only remove containers that don't have the com.cisco.docker-gc.keep set
 	client := helpers.GetDockerClient()
 	project := fmt.Sprintf("com.docker.compose.project=%s", os.Getenv("COMPOSE_PROJECT_NAME"))
 	queryAll := docker.ListContainersOptions{All: true, Filters: map[string][]string{"label": []string{project}}}
 	queryGc := docker.ListContainersOptions{All: true,
-		Filters: map[string][]string{"label": []string{project, "com.lancope.docker-gc.keep"}}}
+		Filters: map[string][]string{"label": []string{project, "com.cisco.docker-gc.keep"}}}
```

Comment:
We still need to keep the `com.lancope.docker-gc.keep` for backwards compatibility in existing repos. I think it makes sense to prefer the `com.cisco` label going forward though (and maybe log a deprecation warning)